platform/android/app/build.gradle.kts

```diff
@@ -487,6 +487,8 @@ android.applicationVariants.all {
             }
             delete(metadataConfig)
             val toArchive = outputsList.filter { file(it).name != "config.lu" } + "config.lu"
+            // Make sure it's not appended to an old file
+            delete(compiledLuaArchive)
```

Comment:
do we still need it after recent changes? https://github.com/coronalabs/corona/commit/82d836f881d632f7100b55f8246a471c21609600#diff-60d531036c673f341a8267c4d40e0545ae42344ce4385abcb7ef5b28cc92fb7b

Reply:
Yes. It's different.

Reply:
I approved this PR. Feel free to merge.

Reply:
Or should I?

Reply:
Yes. Of course.
hah, those are the changes I can made so far.
Thank you.